views/stories.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="card-deck">
+  {{#each stories}}
+  <div class="card">
+    <img class="card-img-top" src="..." alt="Card image cap">
+    <div class="card-body">
+      <h5 class="card-title">{{title}}</h5>
+      <p class="card-text">{{description}}</p>
+    </div>
+    <div class="card-footer">
+      <small class="text-muted">Last updated 3 mins ago</small>
```

Comment:
Maybe remove this, or move the title here

Reply:
Took out the description and just have the title as the footer.